questionnaire/routes.js

```diff
@@ -190,24 +196,19 @@ router
 
             // if the summary section ID is in the progress array, then that means
             // the questionnaire is submittable.
```

Comment:
This comment is still valid and meaningful but is now in the wrong place. Could it be moved to around line 174